gluster-exporter/metric_volume.go

```diff
@@ -206,6 +206,10 @@ func profileInfo() error {
 		return err
 	}
 	for _, volume := range volumes {
+		if value, exists := volume.Options[glusterutils.CountFOPHits]; exists && value == "off" {
```

Comment:
check for `( !exists || value == "off")`

Reply:
Ack